package.json

```diff
@@ -40,45 +36,45 @@
 		"docs:serve": "./node_modules/.bin/http-server docs/jsdoc/",
 		"precommit": "lint-staged",
 		"check:dependencies": "./node_modules/.bin/snyk protect",
-		"prepush": "./node_modules/.bin/eslint $(git ls-files '*.js') && npm run check:dependencies",
+		"prepush":
+			"./node_modules/.bin/eslint $(git ls-files '*.js') && npm run check:dependencies",
 		"prepublishOnly": "npm run check:dependencies"
 	},
 	"dependencies": {
-		"async": "=2.4.1",
+		"async": "=2.6.1",
 		"bignumber.js": "=7.2.1",
-		"bluebird": "=3.5.1",
-		"body-parser": "=1.18.2",
+		"bluebird": "=3.5.3",
+		"body-parser": "=1.18.3",
 		"bytebuffer": "=5.0.1",
-		"change-case": "=3.0.1",
-		"colors": "=1.1.2",
-		"commander": "=2.9.0",
-		"compression": "=1.7.1",
-		"cors": "=2.8.3",
+		"change-case": "=3.0.2",
+		"colors": "=1.3.2",
+		"commander": "=2.19.0",
+		"compression": "=1.7.3",
+		"cors": "=2.8.5",
 		"debug": "=3.1.0",
-		"deep-diff": "=1.0.1",
-		"ejs": "=2.5.6",
-		"express": "=4.16.0",
+		"deep-diff": "=1.0.2",
+		"express": "=4.16.4",
 		"express-domain-middleware": "=0.1.0",
 		"express-query-int": "=1.0.1",
 		"express-rate-limit": "=2.8.0",
-		"extend": "=3.0.1",
-		"find": "=0.2.7",
+		"extend": "=3.0.2",
```

Comment:
We can eliminate `extend` dependency with replacing it with Object.assign

Reply:
Eliminated and replaced with Object.assign calls as we didn't have uses where Object.assign behaves differently.